whipper/command/cd.py

```diff
@@ -551,6 +577,11 @@ def _ripIfNotRipped(number):
 
         self.program.writeLog(discName, self.logger)
 
+        if len(self.skipped_tracks) > 0:
+            logger.warning('%d tracks have been skipped from this rip attempt',
+                           len(self.skipped_tracks))
+            return 5
```

Comment:
Why 5? I understand this is the exit code, but how did you pick 5?

Reply:
5 is arbitrary, I just noticed that 5 wasn't being used for anything else.

Reply:
Maybe exit codes should be documented somewhere user facing (ie., not just in the code)? And maybe it would be better to use powers of two for exit codes so bitmasking could potentially be used to check for multiple error conditions down the line?